locator/local_strategy.cc

```diff
@@ -23,6 +24,9 @@ future<host_id_set> local_strategy::calculate_natural_endpoints(const token& t,
 }
 
 void local_strategy::validate_options(const gms::feature_service&) const {
```

Comment:
nit: `abstract_replication_strategy::validate_options` could provide this default implementation and it could be overriden/called by child classes that need different logic.

Reply:
OK